src/main/java/frc/robot/vision/filters/VisionFilters.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.vision.filters;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.constants.Field;
+import frc.robot.vision.rawdata.AprilTagVisionData;
+import frc.robot.vision.rawdata.VisionData;
+
+public class VisionFilters {
```

Comment:
each one of the filters must be tested multiple times to make sure they are working.